testFunction/saveGame.js

```diff
@@ -0,0 +1,67 @@
+const crawlGameSchedule = require("../utils/crawlGameSchedule");
+const crawlPlayerEntry = require("../utils/crawlPlayerEntry");
+const crawlPlayersInfo = require("../utils/crawlPlayersInfo");
+const Game = require("../models/Game");
+const Player = require("../models/Player");
+
+module.exports = async (dateNumber) => {
+  let dateString = dateNumber;
+  if (typeof date !== "string") {
+    dateString = dateNumber.toString();
+  }
+
+  try {
+    const gameList = await crawlGameSchedule(dateNumber);
+    console.log("get gameList");
+    const players = await crawlPlayerEntry(gameList);
+    console.log("get playerEntry");
+    const playersWithInfo = await crawlPlayersInfo(players);
+    console.log("get playersInfo");
+
+    const newPlayers = [];
+    for (let i = 0; i < playersWithInfo.length; i += 1) {
```

Comment:
```suggestion
    const newPlayers = [];
    
    for (let i = 0; i < playersWithInfo.length; i += 1) {
```
저는 빈줄을 주는 편인데 어떤게 맞는지 모르겠네요..일단 남겨놓겠습니다!

전체 코드 맥락을 보니까 붙이는게 나은 것 같아용